.github/workflows/trigger-publish.yml

```diff
@@ -4,6 +4,7 @@ on:
   push:
     branches:
       - master
+      - release/*
   workflow_dispatch:
```

Comment:
btw, why do we need workflow_dispatch here? what calls this?

Reply:
removed